Formula/icpld.rb

```diff
@@ -0,0 +1,38 @@
+class Icpld < Formula
+  desc "Connection monitor to track your network connection performance."
+  homepage "http://www.ibiblio.org/icpld/"
+  url "http://www.ibiblio.org/icpld/download/icpld-1.1.5.tar.gz"
+  sha256 "9e54c33ce3b2ff2b72f5772aca02ef9f34807b5456054c328eed5b5d61b3a7c1"
+
+  option "with-ipv6", "If you plan on using icpld with IPv6 network."
```

Comment:
These days it could probably just be on by default, probably, unless it causes errors where IPv6 is disabled locally or something?